presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/JdbcPageSink.java

```diff
@@ -161,7 +162,7 @@ else if (DATE.equals(type)) {
             // convert to midnight in default time zone
             long utcMillis = DAYS.toMillis(type.getLong(block, position));
             long localMillis = getInstanceUTC().getZone().getMillisKeepLocal(DateTimeZone.getDefault(), utcMillis);
-            statement.setDate(parameter, new Date(localMillis));
+            statement.setDate(parameter, new Date(localMillis), Calendar.getInstance());
```

Comment:
This is based on discussion found here: https://bugs.mysql.com/bug.php?id=91112

Reply:
Can we also make all the test JDBC URLs (e.g. presto-product-tests/conf/presto/etc/catalog/mysql.properties) look like `jdbc:mysql://mysql:13306?useSSL=false&serverTimezone=UTC` please?

Most 8.0+ instance requires useSSL=true|false, and serverTimezone=??? can address the above Date issue as well.